lib/ChangeTypes.js

```diff
@@ -167,7 +167,7 @@ module.exports = {
   SUBTRACT_FROM_BOTH_SIDES: 'SUBTRACT_FROM_BOTH_SIDES',
   // e.g. 2 = x -> x = 2
   SWAP_SIDES: 'SWAP_SIDES',
-  // e.g. (x - 2) (x + 2) = 0
+  // e.g. (x - 2) (x + 2) = 0 => x = 2, -2
```

Comment:
so the right side will be the node "2, -2"? right now it'd only show one factor right? don't lie here :p